### DISCUSSION TITLE
Wipe tower when restarts After pause

### DISCUSSION THREAD

**Discussion by pollosud** [xblax/flashforge_ad5m_klipper_mod#140] [General]
I want to pause the print then when It restarts to do a wipe tower.is It possibile?

**Comment by xblax** [comment]
That would be something to do in the slicer. You need to enable a wipe tower and and configure pause after layer X, like for manual multi material print filament change. But I've never tried something like that myself.

**Comment by pollosud** [comment]
In klipper can i do a manual comand for pause then wipe tower?

**Reply by xblax** [reply]
No that's not possible or at least I don't know how this could work. A wipe tower must be printed at every layer. That must be defined in the Slicer.

**Comment by pollosud** [comment]
Can i do It with orcaslicer?

**Reply by consp** [reply]
Maybe someone knows, but it would be better to ask that in the orcaslicer discord. See https://github.com/SoftFever/OrcaSlicer/
